pallets/xcm-benchmarks/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "pallet-xcm-benchmarks"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+version = "0.9.27"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false }
+scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
+frame-support = { default-features = false, git = "https://github.com/paritytech/substrate" , branch = "master" }
+frame-system = { default-features = false, git = "https://github.com/paritytech/substrate" , branch = "master" }
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate" , branch = "master" }
+sp-std = { default-features = false, git = "https://github.com/paritytech/substrate" , branch = "master" }
+xcm-executor = {git = "https://github.com/paritytech/polkadot" , branch = "master", default-features = false, features = ["runtime-benchmarks"] }
+frame-benchmarking = { default-features = false, git = "https://github.com/paritytech/substrate" , branch = "master" }
+xcm = { git = "https://github.com/paritytech/polkadot" , branch = "master", default-features = false, features = ["runtime-benchmarks"] }
```

Comment:
I guess we can clean up the feature stuff later, since enabling this feature per default can cause it to be enabled unintentionally.

Reply:
please elaborate

Reply:
Meaning the normal way to enable it is by creating a new feature called `runtime-benchmarks`, and under the feature, we'd also enable all other crates that have a `runtime-benchmarks` feature.